fab/fabfile.py

```diff
@@ -1057,6 +1068,15 @@ def _bower_install(use_current_release=False):
         sudo('bower update --production --config.interactive=false')
 
 
+@parallel
+@roles(ROLES_DJANGO)
+def _npm_install():
+    with cd(env.code_root):
+        sudo('npm prune --production')
+        sudo('npm install --production')
```

Comment:
Oh sorry, I added it back because it *was* necessary. For npm 1.x, install gets js-xpath but update doesn't.